cmd/util/ledger/migrations/migrator_runtime.go

```diff
@@ -25,11 +25,12 @@ func newMigratorRuntime(
 	payloads []*ledger.Payload,
 ) (
 	*migratorRuntime,
+	environment.Accounts,
```

Comment:
I would just add `environment.Accounts` to migrator runtime. Its a collection of runtime useful stuff anyway.